src/main/java/steps/NotePageSteps.java

```diff
@@ -48,16 +47,28 @@ public void setNoteData(Note note) {
         }
     }
 
+    public void clearTitle() {
+        new NotePage().clearTitle();
+    }
+
     public void setTitle(Note note) {
+        new NotePage().setTitle(note.getName());
+    }
+
+    public void updateTitle(Note note) {
```

Comment:
This method is not related to update, it's more about change title.
I suggest you implement one more method for update: 
NotePage notePage = new NotePage();
        String currentTitle = notePage.getNotTitle();
        notePage.setTitle(updateTitle(currentTitle, note.getName()));